Sources/markdown-tool/Commands/PrintHTMLCommand.swift

```diff
@@ -0,0 +1,58 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2023 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import ArgumentParser
+import Markdown
+
+extension MarkdownCommand {
+    /// A command to render HTML for given Markdown content.
+    struct PrintHTML: ParsableCommand {
+        static var configuration = CommandConfiguration(commandName: "print-html", abstract: "Convert Markdown content into HTML")
+
+        @Argument(
+            help: "Input file to print (default: standard input)",
+            completion: .file()
+        )
+        var inputFilePath: String?
```

Comment:
Nit. I see many CLI tool just use `input` to represent `inputFilePath`. 

> Actually, both is fine since most user will use a shorter `-i` version

```suggestion
        var input: String?
```

Reply:
I don't think i can make it print an empty argument label, but i can at least rename the property to `input` or `inputFile` and reword the help text.

Reply:
Here's the result i just pushed:

```
ARGUMENTS:
  <input-file>            Markdown file to print (default: standard input)
```